src/app/lib/utilities/preprocessor/rules/cpstypes.js

```diff
@@ -0,0 +1,27 @@
+import deepGet from '../../../../helpers/json/deepGet';
+
+const whitelist = ['STY', 'MAP'];
+
+const filterUnknownCpsTypes = data => {
+  const groups = deepGet(['content', 'groups'], data);
+
+  if (!groups) {
+    return data;
+  }
+
+  const newGroups = groups.map(g => {
```

Comment:
Also, as clean as it is, can we use words for variables, rather than letters please. Makes it a little bit easier to read quickly